libs/libc/obstack/lib_obstack_object_size.c

```diff
@@ -0,0 +1,41 @@
+/****************************************************************************
+ * libs/libc/obstack/lib_obstack_object_size.c
+ *
+ * This file is a part of NuttX:
+ *
+ *   Copyright (C) 2012 Gregory Nutt. All rights reserved.
+ *   Ported by: Darcy Gong
+ *
+ * It derives from the Rhombus OS math library by Nick Johnson which has
+ * a compatible, MIT-style license:
+ *
+ * Copyright (C) 2009, 2010 Nick Johnson <nickbjohnson4224 at gmail.com>
```

Comment:
Where this license comes from?

Reply:
Copy paste from other lib C file. My fault. Missed that it is not the one from code style.